catalog/spatial/csw/spatial-csw-source/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -204,6 +206,36 @@ public void testFullThread() throws Exception {
         assertMetacard(mc, expectedValues);
 
         expectedValues.clear();
+
+        // verify third metacard's value
```

Comment:
It seems that this code should be moved into its own function since it's used three times.